documentation/release_notes.markdown

```diff
@@ -10,329 +10,158 @@ canonical: "/puppetserver/latest/release_notes.html"
-
-We've also improved the CLI output of `puppet cert list` and `puppet cert sign` to work better with the `--human-readable` and `--machine-readable` flags, and we allow administrators to force a prompt when signing certificates with the `--interactive` flag.
-
-This allows for easier automated failover to authorized nodes within a Puppet infrastructure and provides tools for creating new, securely automated workflows, such as automated component promotions within Puppet-managed infrastructure.
-
--   [SERVER-1305](https://tickets.puppetlabs.com/browse/SERVER-1305)
-
-### Bug fix: Unrecognized parse-opts
-
-Puppet Server 2.4.x used a deprecated API for a Clojure CLI option-parsing library. As a result, calls to `puppetserver gem` (either directly, or indirectly by using a `puppetserver_gem` package resource) generated unexpected warning messages:
-
-    Warning: Could not match Warning: The following options to parse-opts are unrecognized: :flag
-
-Puppet Server 2.5.0 updates this library, which prevents this error message from appearing.
-
--   [SERVER-1378](https://tickets.puppetlabs.com/browse/SERVER-1378)
-
-### Bug fix: Puppet Server no longer ships with an empty PID file
-
-When installed on CentOS 6, Puppet Server 2.4.x included an empty PID file. When running `service puppetserver status`, Puppet Server returned an unexpected error message: `puppetserver dead but pid file exists`.
-
-When performing a clean installation of Puppet Server 2.5.0, no PID file is created, and `service puppetserver status` should return the expected `not running` message.
-
--   [SERVER-1455](https://tickets.puppetlabs.com/browse/SERVER-1455)
--   [EZ-84](https://tickets.puppetlabs.com/browse/EZ-84)
+-   [SERVER-1723](https://tickets.puppetlabs.com/browse/SERVER-1723)
 
 ### Other changes
 
--   [SERVER-1310](https://tickets.puppetlabs.com/browse/SERVER-1310): Error messages in Puppet Server 2.5.0 use the same standard types as other Puppet projects.
--   [SERVER-1121](https://tickets.puppetlabs.com/browse/SERVER-1121): JRuby pool management code for the [Trapperkeeper Webserver Service][Trapperkeeper] is now its own open-source project, [puppetlabs/jrubyutils](https://github.com/puppetlabs/jruby-utils).
+-   [SERVER-1807](https://tickets.puppetlabs.com/browse/SERVER-1807): Puppet Server 5.0 embeds `hocon` gem version 1.2.5 in the default Puppet Server gem path, an upgrade from v1.1.3 used in Server v2.7.2.
+-   [SERVER-1772](https://tickets.puppetlabs.com/browse/SERVER-1772): Added ezbake option to specify additional uberjars to be built and installed with the main project, and allows `puppetserver` to include 2 jars for JRuby 1.7 and JRuby 9k dependencies.
```

Comment:
I think it'd be okay to leave this out of the public release notes.  End users won't care about the ezbake support that we added to enable packaging both JRuby versions and the notes you have above about adding JRuby 9k support seem to cover this fine.